test/src/test/java/hudson/tasks/EnvVarsInConfigTasksTest.java

```diff
@@ -51,6 +55,12 @@ public void setUp() throws Exception {
 		slaveRegular = createSlave(new LabelAtom("slaveRegular"));
 	}
 
+    @Override
```

Comment:
Not going to bother. If you have TABs displayed as four spaces then it looks OK.